tetrateci/eks-ingress.1.8.patch

```diff
@@ -1,3 +1,21 @@
+diff --git a/pkg/test/framework/components/echo/kube/instance.go b/pkg/test/framework/components/echo/kube/instance.go
+index 9b320a8e32..b2d9098218 100644
+--- a/pkg/test/framework/components/echo/kube/instance.go
++++ b/pkg/test/framework/components/echo/kube/instance.go
+@@ -220,10 +220,9 @@ func createVMConfig(ctx resource.Context, c *instance, cfg echo.Config) error {
+ 		if cfg.AutoRegisterVM {
+ 			cmd = append(cmd, "--autoregister")
+ 		}
+-		if !ctx.Environment().(*kube.Environment).Settings().LoadBalancerSupported {
+-			// LoadBalancer may not be suppported and the command doesn't have NodePort fallback logic that the tests do
+-			cmd = append(cmd, "--ingressIP", istiodAddr.IP.String())
+-		}
++
++		cmd = append(cmd, "--ingressIP", istiodAddr.IP.String())
```

Comment:
What address does it return - loadbalancer hostname ip or service ip or nodeip? All the three would work as the call is within the cluster. Ideally we want the loadbalance host ip.

Reply:
host ip